src/app/core/services/courses-list.service.spec.ts

```diff
@@ -1,12 +1,32 @@
 import { TestBed } from "@angular/core/testing";
 
 import { CoursesListService } from "./courses-list.service";
+import { CourseItemModel } from "./models/index";
+import { DEFAULT_CONFIG } from "./courses.config";
 
-describe("CoursesListService", () => {
-  beforeEach(() => TestBed.configureTestingModule({}));
+describe("CoursesListService:", () => {
+  let courseListService: CoursesListService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [CoursesListService],
+    });
+
+    courseListService = TestBed.get(CoursesListService);
+  });
+
+  beforeEach(() => {
+    spyOn(Date, "now").and.returnValue(111111111);
+  });
 
   it("should be created", () => {
-    const service: CoursesListService = TestBed.get(CoursesListService);
-    expect(service).toBeTruthy();
+    expect(courseListService).toBeTruthy();
+  });
+
+  it("should init coursesList$", () => {
+    const coursesList: Array<CourseItemModel> = DEFAULT_CONFIG;
+    courseListService.coursesList$.subscribe(value => {
+      expect(value).toEqual(coursesList);
```

Comment:
So, this way to test observables is not the best.
Adding a delay(1000) in the service, and adding expect(true).toBe(false) you gonna get passed test.
So, to avoid the situation, you can use:
1.) done callback after expect
2.) wrap it callback into async() from angular/core/testing
3.) use marbles - but in the case you will need to add your own scheduler for delay operator and to have a possibility to redefine interval of 1000. In the other way to test delay  with marbles you will need 100 signes "-" in your marbles diagram as far as every "-" is 10ms
4.) use testScheduler